### PR TITLE
Lab results

### PR DESCRIPTION
**Pull request by mmccall**
#### What does this PR do?

Adds back support for Lab Results, makes a few tweaks to it.

#### Related JIRA tickets:

[UPMCFHIR-291](https://jira.amida.com/browse/UPMCFHIR-291)

#### How should this be manually tested?

Run the samples and look at the lab reports, make sure the observations are present.

### CODE REVIEW

File: src/test/java/tr/com/srdc/cda2fhir/CCDTransformerTest.java [hunk 1]
```diff
@@ -217,7 +217,7 @@ public void testSample2() throws Exception {
 		verifySection(bundle, "RESULTS", DiagnosticReport.class, 2, 2);
 		verifySection(bundle, "FUNCTIONAL STATUS", Observation.class, 20, 2);
 		verifySection(bundle, "FAMILY HISTORY", FamilyMemberHistory.class, 1, 1);
-		verifySection(bundle, "MEDICAL EQUIPMENT", Resource.class, 74, 4);
+		verifySection(bundle, "MEDICAL EQUIPMENT", Resource.class, 72, 4);
```
Comment (5t33) [inline src/test/java/tr/com/srdc/cda2fhir/CCDTransformerTest.java]: @mmccall Could you explain why this is decreased?

Comment (5t33) [reply src/test/java/tr/com/srdc/cda2fhir/CCDTransformerTest.java]: performer not added as resource?

**Review by 5t33 — COMMENTED** [review]
do we want to add performer.role? It looks like practitionerRole was removed in R3 and we're still supporting it so we may want to remove it and add performerRole here. It may be a bit a of work to add the valueset to the valueset transformer though.

**Comment by mmccall** [comment]
> do we want to add performer.role? It looks like practitionerRole was removed in R3 and we're still supporting it so we may want to remove it and add performerRole here. It may be a bit a of work to add the valueset to the valueset transformer though.

So the author transformation automatically generates a practitionerRole object with the encoding on it.  It will come in compliant with Healthcare Provider Taxonomy (2.16.840.1.114222.4.11.1066). However in FHIR 4 this is gone, so you would need to map to practitionerRole instead. In FHIR3 practitionerRole can be one of like 6 entries, but in FHIR4 it can be one of those 6, and also a bunch from SNOMED (where concept is healthcare professional), which is what we'd have to map to.

This is a long way of saying we might want to because it would be useful for FHIR4 someday.

**Comment by mmccall** [comment]
this would require mapping the 809 possible codes coming in on CCD to the 460 possible SNOMED codes. I'm not saying it's impossible, but it's a hell of a lot of work.

**Comment by mmccall** [comment]
I just reviewed the main files from Epic, the author on there didn't have a role. It would be good to map, but it isn't appearing and I don't think we are going to have a chance to map the intersection of like 1200 codes. I ticketed it out separately, and we can look back at it later if we want, but I'd say it's orthogonal to this pull request at this point.

**Review by 5t33 — APPROVED** [review]
LGTM

**Pull request merged**